src/test/java/codeu/model/store/persistence/PersistentDataStoreTest.java

```diff
@@ -146,4 +148,77 @@ public void testSaveAndLoadMessages() throws PersistentDataStoreException {
     Assert.assertEquals(contentTwo, resultMessageTwo.getContent());
     Assert.assertEquals(creationTwo, resultMessageTwo.getCreationTime());
   }
+
+  @Test
+  public void testSaveAndLoadEvents() throws PersistentDataStoreException {
+    UUID idOne = UUID.randomUUID();
+    String typeOne = "User";
+    Instant creationOne = Instant.ofEpochMilli(1000);
+    List<String> informationOne = new ArrayList<>();
+    informationOne.add("user_one");
+    Event userEvent = 
+        new Event(idOne, typeOne, creationOne, informationOne);
+
+    UUID idTwo = UUID.randomUUID();
+    String typeTwo = "About Me";
+    Instant creationTwo = Instant.ofEpochMilli(2000);
+    List<String> informationTwo = new ArrayList<>();
+    informationTwo.add("user_two");
+    informationTwo.add("about_me");
+    Event aboutMeEvent = 
+        new Event(idTwo, typeTwo, creationTwo, informationTwo);
+
+    UUID idThree = UUID.randomUUID();
+    String typeThree = "Conversation";
+    Instant creationThree = Instant.ofEpochMilli(3000);
+    List<String> informationThree = new ArrayList<>();
+    informationThree.add("user_three");
+    informationThree.add("conversation_one_title");
+    Event conversationEvent =
+        new Event(idThree, typeThree, creationThree, informationThree);
+
+    UUID idFour = UUID.randomUUID();
+    String typeFour = "Message";
+    Instant creationFour = Instant.ofEpochMilli(4000);
+    List<String> informationFour = new ArrayList<>();
+    informationFour.add("user_four");
+    informationFour.add("conversation_two_title");
+    informationFour.add("message_content");
+    Event messageEvent = 
+        new Event(idFour, typeFour, creationFour, informationFour);
+
+    // save
+    persistentDataStore.writeThrough(userEvent);
+    persistentDataStore.writeThrough(aboutMeEvent);
+    persistentDataStore.writeThrough(conversationEvent);
+    persistentDataStore.writeThrough(messageEvent);
+
+    // load
+    List<Event> resultEvents = persistentDataStore.loadEvents();
+
+    // confirm that what we saved matches what we loaded
+    Event resultEventOne = resultEvents.get(0);
+    Assert.assertEquals(idOne, resultEventOne.getId());
```

Comment:
You could instead use a method like before to compare resultEventOne and userEvent and the others below as well.

Reply:
I also did this to be consistent with the other tests. Do you think I should change it?